sdk/logging.go

```diff
@@ -0,0 +1,94 @@
+package sdk
+
+import (
+	"github.com/sirupsen/logrus"
+)
+
+// Logger provides interface for OPA logger implementations
+type Logger interface {
+	Debug(fmt string, a ...interface{})
+	Info(fmt string, a ...interface{})
+	Error(fmt string, a ...interface{})
+	Warn(fmt string, a ...interface{})
+
+	WithFields(map[string]interface{}) Logger
+	GetFields() map[string]interface{}
+}
+
+// StandardLogger is the default OPA logger
+type StandardLogger struct {
+	fields map[string]interface{}
+}
+
+// NewStandardLogger instantiates new default OPA logger
+func NewStandardLogger() *StandardLogger {
+	return &StandardLogger{}
```

Comment:
To my comment below, if we keep a reference to `*logrus.Logger` (via https://pkg.go.dev/github.com/sirupsen/logrus#New) here, we should be able to control its log level through an additional method on our `Logger` type...

Reply:
Added, have a look and let me know what you think.